### PR TITLE
faws-phase1

### PR DESCRIPTION
**Pull request by viridaria**

### CODE REVIEW

**Comment by catlook** [comment]
@viridaria, review my edits and see if you agree :)

File: docs/index.rst [hunk 1]
```diff
@@ -8,20 +8,20 @@ Fanatical Support for AWS Product Guide
...
+  `https://developer.rackspace.com/docs/fanatical-support-aws <https://developer.rackspace.com/docs/fanatical-support-aws>`_.
+  |vspace|
 
-This Product Guide is designed to provide a detailed look at how Rackspace
+This Product Guide provides a detailed look at how Rackspace
 delivers our **Fanatical Support for AWS** offering. It covers core concepts
 such as the :ref:`AWS account structure <aws_accounts>` and Rackspace
-:ref:`service levels <service_levels>`, and advanced concepts such as
+:ref:`service levels <service_levels>` and advanced concepts such as
 provisioning bastion access via :ref:`Rackspace Passport <passport_v2>` and
```
Comment (horacio3) [inline docs/index.rst]: Passport doesn't use bastion access anymore. Info here: https://docs.rackspace.com/docs/fanatical-support-aws/passport/

advanced concepts such as providing access requests to instances via Passport ...

File: docs/rackspace-elastic-engineering-and-optimizer/index.rst [hunk 2]
```diff
@@ -0,0 +1,228 @@
+.. _rackspace_elastic_engineering_and_optimizer:
+
+=============================================
+Rackspace Elastic Engineering and Optimizer +
+=============================================
+
+Rackspace knows that our customers have varying needs at different stages of
+their cloud journey. That’s why we provide a set of support offers that solve
+your needs at any stage of your cloud lifecycle. Our services provide access to
+the following pwersonnel:
```
Comment (horacio3) [inline docs/rackspace-elastic-engineering-and-optimizer/index.rst]: pwersonnel (Spelling)

Comment (catlook) [reply docs/rackspace-elastic-engineering-and-optimizer/index.rst]: doh!!  Fixed!

**Pull request merged**